### ISSUE TITLE
Keyboard's downListener firing multiple times for Android.

### ISSUE THREAD

**Issue by caiopsouza** [KTXSoftware/Kha#383]
I tested both on my physical device and an emulator. This is different from HTML5 and Flash's behavior, which fires only once even if you hold.

**Comment by RobDangerous** [comment]
Yes, it shouldn't do that. android or android-native? Actual physical keyboard or just a software keyboard?

**Comment by caiopsouza** [comment]
Tested a little more. Both android and android-native behave the same.

Emulator: Soft keyboard works, except backspace which repeats. Menu and back buttons repeat. Physical keyboard repeats both down and up while holding.

Physical device: Soft keyboard works, except backspace which consistently with my keyboard settings (the phone vibrate each time it repeats). Menu doesn't fire when held. Back button repeats if held. Couldn't test physical keyboard.

The code I'm using:

```haxe
Keyboard.get().notify
(
	function(x, y) trace("down", x, y),
	function(x, y) trace("up", x, y)
);
Keyboard.get().show();
```
